src/components/common/WalletInfo/index.tsx

```diff
@@ -61,7 +61,7 @@ export const WalletInfo = ({
   return (
     <Box className={css.container}>
       <Box className={css.accountContainer}>
-        <ChainIndicator />
```

Comment:
Should we add a small testcase for this fix to the WalletInfo unit test?

Reply:
It will be redesigned in #2678, so I wouldn't at this time.

Reply:
Makes sense. Vamos then :)